website/pages/fdc3-roadmap.html

```diff
@@ -0,0 +1,384 @@
+        alt="GitHub issue custom search in repo"></a></td>
+  </tr>
+
+
+  <tr>
+    <th>Closed
+    <td><a
+      href="https://github.com/finos/FDC3/issues?q=is%3Aclosed%20is%3Aissue%20milestone%3A%222.0%22%20label%3Acontext-data"><img
+        src="https://img.shields.io/github/issues-search/finos/fdc3?label=context-data%20issues&amp;query=is%3Aclosed%20is%3Aissue%20milestone%3A%222.0%22%20label%3Acontext-data"
+        alt="GitHub issue custom search in repo"></a></td>
+    <td><a
+      href="https://github.com/finos/FDC3/issues?q=is%3Aclosed%20is%3Aissue%20milestone%3A%222.0%22%20label%3Aapi"><img
+        src="https://img.shields.io/github/issues-search/finos/fdc3?label=api%20issues&amp;query=is%3Aclosed%20is%3Aissue%20milestone%3A%222.0%22%20label%3Aapi"
+        alt="GitHub issue custom search in repo"></a></td>
+    <td><a
+      href="https://github.com/finos/FDC3/issues?q=is%3Aclosed%20is%3Aissue%20milestone%3A%222.0%22%20label%3Aapp-directory"><img
+        src="https://img.shields.io/github/issues-search/finos/fdc3?label=app-directory%20issues&amp;query=is%3Aclosed%20is%3Aissue%20milestone%3A%222.0%22%20label%3Aapp-directory"
+        alt="GitHub issue custom search in repo"></a></td>
+    <td><a
+      href="https://github.com/finos/FDC3/issues?q=is%3Aissue+is%3Aclosed+label%3A&quot;formal+specification&quot;+milestone%3A&quot;2.0&quot;"><img
+        src="https://img.shields.io/github/issues-search/finos/fdc3?label=formal-specification%20issues&amp;query=is%3Aissue+is%3Aclosed+label%3A&quot;formal+specification&quot;+milestone%3A&quot;2.0&quot;"
+        alt="GitHub issue custom search in repo"></a></td>
+  </tr>
+
+
+
+  <tr>
+    <th colspan="2">Issues Complete</th>
+    <td>soon</td>
+    <td></td>
+    <td></td>
+    <td>soon</td>
+  </tr>
+  <tr>
+    <th colspan="2">PRs Accepted By Working Group</th>
+    <td colspan="4">26th May</td>
+  </tr>
+  <tr>
+    <th colspan="2">PRs Merged into Pre-Draft</th>
+    <td colspan="4"></td>
+  </tr>
+
+
+  <tr>
+    <th colspan=6><h3>Adoption</h3></th>
+  </tr>
+  <tr>
+    <th colspan="2">Propose Pre-Draft</th>
+    <td colspan=4>Before June 23rd</td>
+  </tr>
+  <tr>
+    <th colspan="2">Vote</th>
+    <td colspan=4>June 23rd - June 30th <em>By Consensus,
+        Determined By Maintainters</em></td>
+  </tr>
+  <tr>
+    <th colspan="2">Adopted Pre-Draft</th>
+    <td colspan=4>July 1st</td>
+  </tr>
+  <tr>
+    <th colspan="2">Announce Adopted FDC 2.0 Draft</th>
+    <td colspan="4">July 13th</td>
+  </tr>
+  <tr>
+    <th colspan="2">45 day wait <em>check for prior art, patents</em></th>
+    <td colspan="4"></td>
+  </tr>
+  <tr>
+    <th colspan="2">FDC 2.0 Released</th>
+    <td colspan=4>FDC 2.0, <em>Mid September</em></td>
+  </tr>
+  <tr>
+    <th colspan="2">Adopt Community Spec License</th>
+    <td colspan="6">Mid September</td>
```

Comment:
I think we can do this shortly after pre-draft is adopted, don't have to wait for the release. That way anything contributed in the interim is on CSL rather than IP Policy.

Reply:
Should we bring this forward then?  The reason we chose mid-september is to give it the 45 day window.  I can change this to Mid August or something?